spec/factories/users.rb

```diff
@@ -2,32 +2,37 @@
 FactoryGirl.define do
   factory :user do
     username
-    email
     password '12345678'
-    before(:create) { |user| user.as_f_member }
-
-    # Overrides the :create_profile_for_user method
-    after(:build) { |user| user.class.skip_callback(:create, :after, :create_profile_for_user) }
-
-    # Uses profiles factory to create a valid profile
-    after(:create) do |user, evaluator|
-      create(:profile, user: user)
-    end
+    firstname
+    lastname
+    email
+    phone
+    stil_id
+    confirmed_at { Time.zone.now }
+    member_at { Time.zone.now }
 
-    # Can be called to create a user with the :create_profile_for_user method
-    trait :user_with_create_profile do
-      after(:create) { |user| user.send(:create_profile_for_user) }
+    trait :admin do
+      password '12345678'
+      with_admin_post
     end
-
-    # Needed because users cannot be created without f_validate being okay.
-    to_create { |instance| instance.save!(validate: false) }
   end
 
-  factory :admin, class: 'User' do
+  factory :admin, class: 'User' do |user|
```

Comment:
Unused block argument - `user`. You can omit the argument if you don't care about it.